server/server_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"github.com/mattermost/mattermost-push-proxy/internal/version"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
 	"github.com/stretchr/testify/assert"
 	"net/http"
 	"net/http/httptest"
```

Comment:
Everything is messed up here. All standard library packages should be in one group, then push-proxy packages in another group, and all third party in another group.